main.py

```diff
@@ -1,12 +1,253 @@
+    total_inventory=input("Enter inventory: ")
+
+    request_body = {
+            "title":title,
+            "release_date":release_date,
+            "total_inventory":total_inventory,
+            }
+    
+    r = requests.post(f"{URL}/videos", json=request_body)
+    
+    if r.status_code != 201:
+        print(r.text)
+    else:
+        print("Add Successful!")
+#2
+def edit_a_video():
+    video=input("\nEnter Video Id: ")
+    title=input("Enter a title: ")
+    release_date=input("Enter a release date: ")
+    total_inventory=input("Enter inventory: ")
+
+    request_body = {
+            "title":title,
+            "release_date":release_date,
+            "total_inventory":total_inventory,
+            }
+    
+    r = requests.put(f"{URL}/videos/{video}", json=request_body)
+
+    if r.status_code != 200:
+        print(r.text)   # explain?
+    else:
+        print("Edit Successful!")
+
+#3
+def delete_a_video():
+    video=input("\nEnter Video Id: ")
+    
+    r = requests.delete(f"{URL}/videos/{video}")
+
+    if r.status_code != 200:
+        print(r.text)
+    else:
+        print("Delete Video Successful!")
+
+#4
+def get_info_all_videos():
+    r = requests.get(f"{URL}/videos")
+
+    if r.status_code != 200:
+        print(r.text)   
+    else:
+        print(r.json()) 
+
+#5
+def get_info_video():
+    video=input("\nEnter Video Id: ")
+
+    r = requests.get(f"{URL}/videos/{video}")
+
+    if r.status_code != 200:
+        print(r.text)   
+    else:
+        print(r.json()) 
+
+#6
+def add_a_cutomer():
+    name=input("\nEnter a customer name: ")
+    postal_code=input("Enter postal code: ")
+    phone=input("Enter phone number: ")
+
+    request_body = {
+            "name":name,
+            "postal_code":int(postal_code),
+            "phone":phone,
+            }
+    
+    r = requests.post(f"{URL}/customers", json=request_body)
+    
+    if r.status_code != 201:
+        print(r.text)
+    else:
+        print("✅ 👍 Add Successful!")
+#7
+def edit_a_customer():
+    customer_id=input("\n Enter Customer ID: ")
+    name=input("Enter a customer: ")
+    postal_code=input("Enter postal code: ")
+    phone=input("Enter phone number: ")
+
+    request_body = {
+            "name":name,
+            "postal_code":int(postal_code),
+            "phone":phone,
+            }
+    
+    r = requests.put(f"{URL}/customers/{customer_id}", json=request_body) #???
+
+    if r.status_code != 200:
+        print(r.text)   # explain?
+    else:
+        print("Edit Successful!")
+# 8
+def delete_a_customer():
+    customer_id=input("\n Enter Customer ID: ")
+
+    r = requests.delete(f"{URL}/customers/{customer_id}")
+
+    if r.status_code != 200:
+        print(r.text)
+    else:
+        print("Delete Customer Successful!")   
+#9
+def get_info_one_customer():
+    r = requests.get(f"{URL}/customers")
+    if r.status_code != 200:
+        print(r.text)   
+    else:
+        print(r.json()) 
```

Comment:
This gets info about every customer.  It should prompt for a customer like `delete_a_customer` does.